starwars/src/components/CharacterList.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import CharacterCard from './CharacterCard';
+
+
+
+const CharacterList = (props) => {
+    return (
+        <div>
+            {props.characters.map(characters => (<CharacterCard characters={characters} />))}
```

Comment:
When using map functions, it may give you more clarity to use singular nouns for the elements _inside_ the map function. Just so you now you're only dealing with one object. 
```props.characters.map(character => (<CharacterCard character={character} />))```
is more descriptive of what's actually happening.